pkg/ibmvpc/parser.go

```diff
@@ -708,14 +725,20 @@ func getTgwObjects(c *datamodel.ResourcesContainerModel,
 			continue
 		}
 		if _, ok := tgwMap[tgwUID]; !ok {
+			region := ""
+			if ok { // if there is a transit gateway in the config file
```

Comment:
this `ok` (line `729`) refers the one defined from line `727` instead of the one you meant (from line `693`)